packages/client/src/pages/Profile/Profile.tsx

```diff
@@ -1,3 +1,21 @@
-export function Profile() {
-  return <h1>Profile</h1>
+import { FC, useState } from 'react'
+import styles from './profile.module.css'
+import ProfileInfo from './elements/ProfileInfo'
+import ProfilePassword from './elements/ProfilePassword'
+import { TabProfile } from './type'
+import ProfileExit from './elements/ProfileExit'
+
+export const Profile: FC = () => {
+  const [currentTab, setCurrentTab] = useState<TabProfile>(
+    TabProfile.profileInfo
+  ) // todo: переключение между вкладками будет работать после подключения store
+  return (
+    <div className={styles.container}>
+      {currentTab === TabProfile.profileInfo && <ProfileInfo />}
+      {currentTab === TabProfile.profilePassword && <ProfilePassword />}
+      {currentTab === TabProfile.profileExit && <ProfileExit />}
```

Comment:
[Предложение]
Здесь все условия взаимноисключающие, но с таким условным рендерингом эта логика теряется.

Тут можно использовать отдельный компонент или функцию рендера, если считаешь, что на компонент не тянет, со `switch`. Но мне больше нравится делать объект вне компонента, чтобы не было пересозданий при ререндере, вот так:
```ts
const tabContent: Record< TabProfile, ReactNode> = {
  [TabProfile.profileInfo]: <ProfileInfo />,
  [TabProfile.profilePassword]: <ProfilePassword />,
  [TabProfile.profileExit]: <ProfileExit />
}
``` 

И тогда тут можно будет просто взять значение из объекта
```suggestion
      {tabContent[currentTab]}
```